nist/system/ksp-block-nist-au-8-1-timestamps-sync-time-zone.yaml

```diff
@@ -0,0 +1,30 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-nist-au-8-1-timestamps-sync-time-zone
+  namespace: default
+spec:
+  tags: ["NIST", "AU-8-1", "LINUX", "TIMESTAMPS"]
+  message: "Alert! editing timezone files are not allowed!"
```

Comment:
`editing` -> `Editing`

Reply:
@raviknox Thank you for your review. Policy has been modified.